integration_tests/CMakeLists.txt

```
@@ -270,6 +270,8 @@ RUN(NAME include_01 LABELS gfortran llvm llvm_wasm llvm_wasm_emcc wasm c fortran
 RUN(NAME include_02 LABELS gfortran llvm llvm_wasm llvm_wasm_emcc wasm c fortran)
 RUN(NAME include_03 LABELS gfortran llvm llvm_wasm llvm_wasm_emcc wasm c INCLUDE_PATH include_03 fortran)
 
+RUN(NAME use_01 LABELS gfortran llvm llvm_wasm llvm_wasm_emcc wasm c fortran)
```

The C test does not work for this.
```suggestion
RUN(NAME use_01 LABELS gfortran llvm llvm_wasm llvm_wasm_emcc wasm fortran)
```

---

Yes, it was an issue with the C back-end. I have a PR open for this at #4199.